lib/src/map_widget.dart

```diff
@@ -186,8 +220,8 @@ class _MapWidgetState extends State<MapWidget> {
       'mapboxPluginVersion': '1.0.0-beta.2'
     };
 
-    return _mapboxMapsPlatform.buildView(
-        creationParams, onPlatformViewCreated, widget.gestureRecognizers);
+    return (_mapboxMapsPlatform.buildView(widget.androidHostingMode,
+        creationParams, onPlatformViewCreated, widget.gestureRecognizers) as PlatformViewLink).createState().;
```

Comment:
Can be either `PlatformViewLink` or `AndroidView` - force cast could fail right?

Reply:
Thanks for catching this! Remnants from some debugging, removed.